src/cljs/chronograph_web/routes.cljs

```diff
@@ -0,0 +1,26 @@
+(ns chronograph-web.routes
+  (:require [reagent.dom]
+            [re-frame.core :as rf]
+            [bidi.bidi :as bidi]
+            [pushy.core :as pushy]
+            [chronograph-web.pages.landing.views :refer [landing-page]]
+            [chronograph-web.pages.create-organization.views :refer [create-organization-page]]
+            [chronograph-web.pages.organization.views :refer [organization-page]]
+            [chronograph-web.events.routing :as routing-events]))
+
+(def routes ["/" {"" :root
+                  "organization/" {"new" :organization-new
+                                   [:slug] :organization-show}}])
+
+(def authenticated-view {:root landing-page
+                         :organization-new create-organization-page
+                         :organization-show organization-page})
+
+(defn set-page! [match]
+  (rf/dispatch-sync [::routing-events/set-page match]))
```

Comment:
Why does this need to be sync? It should be async, since it can potentially fire when interacting with the UI.

Reply:
This was also what was preventing pushy from routing when set-token was called. This is fixed.